syntax = "proto3";
package jio.brain.proto.knowledge.healthcare;

import "jio/brain/proto/knowledge/healthcare/base/status.proto";

message GetSymptomGivenDiseaseRequest {
    repeated uint32 disease_id = 1;
}

// Returns a list of Symptom IDs
message GetSymptomGivenDiseaseResponse { 
    repeated uint32 symptom_id = 1;
    jio.brain.proto.knowledge.healthcare.base.Status status = 2;
}
